versions.yaml

```diff
@@ -190,6 +190,8 @@ externals:
     # instead of abstract socket, thus kata wouldn's support the containerd's
     # version older than them.
     version: "v1.5.2"
+    # CCv0 is using a PR containerd build as our changes are still PoC
+    pr_id: "5911"
```

Comment:
Could we push the `tools` and `versions.yaml` changes at least as a separate commit, please :)

Reply:
I've created PR #2577 for this change